gdrive/sheets_client.py

```diff
@@ -221,17 +222,23 @@ def upload_participant(
     ]
 
     body = {"values": values}
-    result = (
-        sheets_service.spreadsheets()
-        .values()
-        .append(
-            spreadsheetId=settings.SHEETS_ID,
-            range="Sheet1!A1",
-            valueInputOption="RAW",
-            body=body,
+
+    try:
+        result = (
+            sheets_service.spreadsheets()
+            .values()
+            .append(
+                spreadsheetId=settings.SHEETS_ID,
+                range="Sheet1!A1",
+                valueInputOption="RAW",
+                body=body,
+            )
+            .execute()
         )
-        .execute()
-    )
-    if "error" in result:
```

Comment:
Is there some part of result that should be logged in non error case?

Reply:
Make sheet name an setting?

Reply:
> Is there some part of result that should be logged in non error case?

Non error case is logging in export API

Reply:
> Make sheet name an setting?

Set up so that the setting can be optionally populated